<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PlayerSettingsUI</class>
 <widget class="QWidget" name="PlayerSettingsUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>407</width>
    <height>178</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QRadioButton" name="cbSoundSystem">
     <property name="text">
      <string>Use the &amp;KDE sound system</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="volumeHolder" native="true">
     <layout class="QGridLayout">
      <property name="margin">
       <number>1</number>
      </property>
      <item row="1" column="4">
       <widget class="QLabel" name="textLabel3">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>100%</string>
        </property>
       </widget>
      </item>
      <item row="1" column="3">
       <spacer>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Expanding</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>261</width>
          <height>16</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="1" column="2">
       <widget class="QLabel" name="textLabel2">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>0%</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QCheckBox" name="cbVolume">
        <property name="text">
         <string>Force &amp;Volume:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2" colspan="3">
       <widget class="QSlider" name="volumeSlider">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="maximum">
         <number>100</number>
        </property>
        <property name="singleStep">
         <number>10</number>
        </property>
        <property name="value">
         <number>100</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="0" column="0" rowspan="2">
       <spacer>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Fixed</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>41</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QRadioButton" name="cbExternal">
     <property name="text">
      <string>&amp;Use an external player</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout">
     <property name="margin">
      <number>0</number>
     </property>
     <item>
      <spacer>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="textLabel5">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>&amp;Player:</string>
       </property>
       <property name="buddy">
        <cstring>reqExternal</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="KUrlRequester" name="reqExternal" native="true">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
         <horstretch>10</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QRadioButton" name="cbNone">
     <property name="text">
      <string>&amp;No audio output</string>
     </property>
    </widget>
   </item>
   <item>
    <spacer>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KUrlRequester</class>
   <extends>QWidget</extends>
   <header>kurlrequester.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>cbExternal</sender>
   <signal>toggled(bool)</signal>
   <receiver>textLabel5</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>91</x>
     <y>98</y>
    </hint>
    <hint type="destinationlabel">
     <x>67</x>
     <y>128</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cbExternal</sender>
   <signal>toggled(bool)</signal>
   <receiver>reqExternal</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>240</x>
     <y>98</y>
    </hint>
    <hint type="destinationlabel">
     <x>312</x>
     <y>128</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cbSoundSystem</sender>
   <signal>toggled(bool)</signal>
   <receiver>volumeHolder</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>71</x>
     <y>13</y>
    </hint>
    <hint type="destinationlabel">
     <x>11</x>
     <y>35</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cbVolume</sender>
   <signal>toggled(bool)</signal>
   <receiver>textLabel2</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>72</x>
     <y>40</y>
    </hint>
    <hint type="destinationlabel">
     <x>149</x>
     <y>60</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cbVolume</sender>
   <signal>toggled(bool)</signal>
   <receiver>volumeSlider</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>124</x>
     <y>39</y>
    </hint>
    <hint type="destinationlabel">
     <x>178</x>
     <y>50</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cbVolume</sender>
   <signal>toggled(bool)</signal>
   <receiver>textLabel3</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>38</x>
     <y>39</y>
    </hint>
    <hint type="destinationlabel">
     <x>380</x>
     <y>64</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
